polar-core/src/numerics.rs

```diff
@@ -300,46 +314,46 @@ mod tests {
         assert!(Numeric::Integer(-1) != Numeric::Float(1.0));
         assert!(Numeric::Integer(2) > Numeric::Float(1.0));
         assert!(Numeric::Integer(-2) < Numeric::Float(1.0));
-        assert!(Numeric::Integer(1 << 52) == Numeric::Float((2.0_f64).powi(52)));
-        assert!(Numeric::Integer(1 << 53) == Numeric::Float((2.0_f64).powi(53)));
-        assert!(Numeric::Integer((1 << 52) + 1) == Numeric::Float((2.0_f64).powi(52) + 1.0));
-        assert!(Numeric::Integer(1 << 52) < Numeric::Float((2.0_f64).powi(52) + 1.0));
-        assert!(Numeric::Integer((1 << 52) + 1) > Numeric::Float((2.0_f64).powi(52)));
-        assert!(Numeric::Integer(-(1 << 52) - 1) < Numeric::Float(-(2.0_f64).powi(52)));
+        assert!(Numeric::Integer(1 << 52) == Numeric::Float((2.0 as f64).powi(52)));
```

Comment:
This change should be reverted. This is undoing an improvement we made since the original branch